cwatch.php

```diff
@@ -685,29 +705,31 @@ private function getMalwareTab($template, $service, $post)
                     'protocol' => $post['port'] == '22' ? 'FTPS' : 'FTP'
                 ]
             );
-            $this->log('addmalwarescanner', serialize($scanner), 'input', empty($scanner->errorMsg));
+            $last_request = $api->lastRequest();
+            $last_request['password'] = isset($last_request) ? '***' : '';
```

Comment:
I thought the last request only contains an array of 'content' and 'headers' keys? You're adding a 'password' key but not hiding the password in the content?